### DISCUSSION TITLE
winget installed a program successfully but the program still not executable in terminal

### DISCUSSION THREAD

**Discussion by Squirre17** [microsoft/winget-cli#2825] [Q&A]
```powershell
prompt> winget install LLVM.LLVM
Found an existing package already installed. Trying to upgrade the installed package...
No applicable upgrade found.

```
I have installed llvm several times and it return several times successful message.

but llvm or clang still can't execute
```powershell
prompt> llvm
llvm: The term 'llvm' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

my try:
add option `--force`, close and reopen terminal, run terminal as admin..... but no one work, That so annoying me.

**Comment by xarthurx** [comment]
By default it will not add to `PATH`.
You need to do `winget install -i LLVM.LLVM` and manually select to add it to your `PATH`.
